pkg/webhook/mutation/pod/common/metadata/annotations.go

```diff
@@ -26,11 +26,13 @@ func copyAccordingToPrefix(pod *corev1.Pod, namespace corev1.Namespace) map[stri
 
 	for key, value := range namespace.Annotations {
 		if strings.HasPrefix(key, dynakube.MetadataPrefix) {
-			added := setPodAnnotationIfNotExists(pod, key, value)
+			setPodAnnotationIfNotExists(pod, key, value)
+		}
+	}
 
-			if added {
-				addedAnnotations[key] = value
-			}
+	for key, value := range pod.Annotations {
+		if strings.HasPrefix(key, dynakube.MetadataPrefix) {
+			addedAnnotations[key] = value
```

Comment:
the name `addedAnnotations` is a bit misleading now, could be just `metadataAnnotations` or something similar

Reply:
done 👍